platform/device-manager.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * DeviceManager Stub
+ * TODO(cjdaly) integrate with Mock Local Home Platform and Mock Network
+ **/
+/// <reference types="@google/local-home-sdk" />
+
+export const DeviceManagerStub = class {
```

Comment:
should this extend the DeviceManager type from the platform?

Reply:
Yes.  I added the implementation, though am leaving the methods empty for now.  I'll fill in the functionality for them in a later PR #23